src/d3a/models/area/__init__.py

```diff
@@ -70,24 +71,16 @@ def __init__(self, name: str = None, children: List["Area"] = None,
                  grid_fee_percentage: float = None,
                  grid_fee_constant: float = None,
                  external_connection_available: bool = False,
-                 baseline_peak_energy_import_kWh: float = None,
-                 baseline_peak_energy_export_kWh: float = None,
-                 import_capacity_kVA: float = None,
-                 export_capacity_kVA: float = None
+                 throughput: ThroughputParameters = None
                  ):
         validate_area(grid_fee_constant=grid_fee_constant,
-                      grid_fee_percentage=grid_fee_percentage,
-                      baseline_peak_energy_import_kWh=baseline_peak_energy_import_kWh,
-                      baseline_peak_energy_export_kWh=baseline_peak_energy_export_kWh,
-                      import_capacity_kVA=import_capacity_kVA,
-                      export_capacity_kVA=export_capacity_kVA)
+                      grid_fee_percentage=grid_fee_percentage)
         self.balancing_spot_trade_ratio = balancing_spot_trade_ratio
         self.active = False
         self.log = TaggedLogWrapper(log, name)
         self.current_tick = 0
         self.name = name
-        self.baseline_peak_energy_import_kWh = baseline_peak_energy_import_kWh
-        self.baseline_peak_energy_export_kWh = baseline_peak_energy_export_kWh
+        self.throughput = ThroughputParameters() if throughput is None else throughput
```

Comment:
Following this https://github.com/gridsingularity/d3a/pull/995/files#r532533493
```suggestion
        self.throughput = throughput
```